static/app/components/stories/jsxProperty.tsx

```diff
@@ -34,6 +36,9 @@ export default function JSXProperty({name, value}: Props) {
     );
   }
   if (isValidElement(value)) {
+    if (value.type === JSXNode) {
+      return <code data-property="element">{[`${name}={`, value, '}']}</code>;
+    }
```

Comment:
Tiny bugfix for `JSXProperty` display. Previously, something like `{actions: <JSXNode name="Button" />}` would render as `actions=[Object object]` but now renders as `actions={<Button />}`

cc @ryan953